image_tools/src/cam2image.cpp

```diff
@@ -113,9 +114,10 @@ int main(int argc, char * argv[])
   // parameter.
   custom_camera_qos_profile.history = history_policy;
 
+  std::cout << "Publishing data on topic " << topic << std::endl;
```

Comment:
Huh, OK, I didn't know about that piece of language from POSIX.  I've switched it to use printf now.